search/django/weedid/tasks.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import, unicode_literals
+from celery import shared_task
+from weedcoco.repo.deposit import deposit
+from weedcoco.index.indexing import ElasticSearchIndex
+from weedcoco.index.thumbnailing import thumbnailing
+from weedid.models import Dataset
+from core.settings import THUMBNAILS_DIR, REPOSITORY_DIR, DOWNLOAD_DIR
+from pathlib import Path
+
+
+@shared_task
+def upload_task(weedcoco_path, image_dir, upload_id):
```

Comment:
This is performed _after_ uploading, so let's call it `finish_upload_task` or `submit_upload_task`?